sycl/include/sycl/ext/intel/experimental/esimd/detail/operators.hpp

```diff
@@ -301,8 +300,7 @@ namespace __SEIEE {
             class = std::enable_if_t<                                          \
                 __SEIEED::is_any_simd_view_type_v<SimdViewT1> && COND>>        \
   inline auto operator BINOP(const SimdViewT1 &LHS, T2 RHS) {                  \
-    using SimdT = typename SimdViewT1::value_type;                             \
-    return LHS.read() BINOP SimdT(RHS);                                        \
+    return LHS.read() BINOP RHS;                                               \
```

Comment:
Similar issues need to be fixed for CMPOP implementation below.

Reply:
> Similar issues need to be fixed for CMPOP implementation below.

I suggest to do this in another PR.

Reply:
Yes, CMPOP needs a similar fix. I will do this change in a separate PR.